flink-clients/src/main/java/org/apache/flink/client/cli/CliFrontend.java

```diff
@@ -561,7 +640,7 @@ protected int cancel(String[] args) throws Exception {
 	 *
 	 * @param args Command line arguments for the cancel action.
 	 */
-	protected int savepoint(String[] args) throws CliArgsException {
+	protected void savepoint(String[] args) throws Exception {
```

Comment:
Why was this changed? It's the only command which doesn't return a status code. The diff would have been smaller without it.

Reply:
I forgot to remove the return codes from the other methods. Will change it.